projects/cpython3/gen_fuzz.py

```diff
@@ -0,0 +1,47 @@
+"""Generate a fuzz test.
```

Comment:
> Also, it'd be "dead code" in cpython's repo as it would never be executed

That's the point. We encourage everyone to *not* have fuzz targets as dead code, and instead use them as part of CI
(w/o fuzzing): see https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md

> I guess I am terrified of #include SOME_CONSTANT

Yea, don't do that. 

Also, why do you have this two-level complexity where you first define e.g. `fuzz_builtin_unicode`
and then call it in LLVMFuzzerTestOneInput. 

The way we propose is to just have LLVMFuzzerTestOneInput in your code for every API you want to test, 
see e.g. https://github.com/google/boringssl/tree/master/fuzz
Then you won't need this python script, and it will be very easy to incorporate the fuzz targets into the upstream build system
(e.g. using  https://github.com/llvm-mirror/llvm/tree/master/lib/Fuzzer/standalone or similar)